wpeverest/everest-forms
# Field - Date and time field enhancement

### All Submissions:






### Changes proposed in this Pull Request:

This PR includes the enhancement of date and time field. Date Localization, Min/Max date, and multiple date selection features are introduced. 

### How to test the changes in this Pull Request:

1. Create a new form or edit the previous one.
2. Drag the date and field and you can see the new feature options under Advanced options.
3. After you made your desired changes save the form and see the frontend.

### Types of changes:




### Other information:




### Changelog entry

> Feature - Date localization.
> Feature - Min/Max Date.
> Feature - Multiple date selection.
